README.md

```diff
@@ -89,10 +89,11 @@ The Diamond Bluff project is being created to address these questions and to
 foster the emergence of such an open and creative software eco-system for
 DPU/IPU based cloud infrastructure. The project intends to delineate what a
 DPU/IPU is, to loosely define a framework(s) and architecture for a
-DPU/IPU-based software stack(s) applicable to any vendors hardware solution, to
-integrate with existing open source projects aligned to the same vision such as
-the Linux kernel and[ IPDK.io](https://ipdk.io) and create new APIs for
-interaction with and between the elements of the DPU/IPU ecosystem:
+DPU/IPU-based software stack(s) applicable to any vendors hardware solution,
+allowing the creation of a rich open source application ecosystem, to integrate
+with existing open source projects aligned to the same vision such as the Linux
+kernel and[ IPDK.io](https://ipdk.io) and create new APIs for interaction with
```

Comment:
Another tweak (in a section you didn't change!) - "and to create new APIs ...."

Reply:
Sure, I'll clean this up as a part of this commit.